cognite/neat/_rules/models/dms/_rules_input.py

```diff
@@ -319,7 +319,7 @@ def referenced_views_and_containers(self) -> tuple[set[ViewEntity], set[Containe
         return views, containers
 
     def as_view_entities(self) -> list[ViewEntity]:
-        return [view.as_entity_id(self.metadata.space, self.metadata.external_id) for view in self.views]
+        return [view.as_entity_id(self.metadata.space, self.metadata.version) for view in self.views]
```

Comment:
wouldn't the id consist of space, external id and version ?

Reply:
Fixed in later PR.